src/app/custom-styles.css

```diff
@@ -139,3 +150,17 @@ main:has(.hide-footer-bar) ~ .footer-bar::before {
 main:has(.hide-footer-bar) ~ .footer-bar {
   display: none !important;
 }
+
+.site-background {
+  background: var(--background);
+  background-image: url('/images/footer-bg.png');
```

Comment:
(Already mentioned but posting here for visibility) Moving the image from the CTA to this class and applying it to the top level `div` seems to cause the responsive design to break. Here's an example when the screen's width gets smaller:
<img width="640" alt="image" src="https://github.com/user-attachments/assets/7c8cfd71-22eb-420a-8be7-8267e9dc7007" />